components/lv_examples/lv_ex_conf.h

```diff
@@ -16,7 +16,7 @@
 /*******************
  * GENERAL SETTING
  *******************/
-#define LV_EX_PRINTF       1       /*Enable printf-ing data in demoes and examples*/
+#define LV_EX_PRINTF       CONFIG_LV_EX_PRINTF       /*Enable printf-ing data in demoes and examples*/
```

Comment:
(This applies to all the changes in `lv_ex_conf.h`.)

As far as I know, Kconfig does not define unselected options to 0; it just doesn't define them at all. I'm not sure how the statement `#if LV_EX_PRINTF` will be affected by this. It might be better to use a code pattern like this:

```suggestion
/*Enable printf-ing data in demoes and examples*/
#ifdef CONFIG_LV_EX_PRINTF
#define LV_EX_PRINTF       1
#else
#define LV_EX_PRINTF       0
#endif
```

Reply:
You're totally right @embeddedt, I forgot about that, will commit your suggestion.

Reply:
This is the pattern I use in the new lv_conf_internal.h:
https://github.com/lvgl/lvgl/blob/1c96215db9ec9d3de0dd10c2403e124cbb4a8dd6/src/lv_conf_internal.h#L107

Reply:
I'm not sure but I think we would need to include the ESP-IDFs `sdkconfig.h` file to be able to use the CONFIG_x symbols. I will try removing that include on the current `lv_config` file and see what happens.

Reply:
If it's needed we can use [LV_CONF_PATH](https://github.com/lvgl/lvgl/blob/1c96215db9ec9d3de0dd10c2403e124cbb4a8dd6/src/lv_conf_internal.h#L15) to include any file instead of `lv_conf.h`.

Reply:
Nice, I will test it too, yesterday I forgot to push the fixes for @embeddedt comment, will do it later.